cmd/clusterctl/client/cluster/mover_test.go

```diff
@@ -392,6 +392,30 @@ var moveTests = []struct {
 			},
 		},
 	},
+	{
```

Comment:
Added an additional external object test with a global resource. Also note that I had to change the tests a bit below to look across all namespaces instead of just ns1

Reply:
might be I'm wrong, but I don't see the test with a namespaced resource

Reply:
updated with test for namespaced  + global external objects. going to resolve these convos.